### PR TITLE
Add buffer import to simplify browser builds

### LINKED ISSUE #403: global Buffer usage complicates browser usage
Accessing Buffer as a global works great in Node.js, but in the browser requires each build tool to inject that global in its own special way.

Would you be open to a small PR that uses import/require to access Buffer to simplify browser build issues?

### PR DESCRIPTION
**Pull request by JAForbes**
Fixes #403

### CODE REVIEW

**Comment by afzaal-ameer** [comment]
@JAForbes I haven't tested this, but I see references to both `buffer` and `Buffer` in some files. e.g. https://github.com/JAForbes/avsc/blob/52d2b93be59af6f5228599399c0881aad25e70e0/lib/index.js#L69

File: etc/benchmarks/avro-serialization-implementations/scripts/decode/node-avro-io.js [hunk 1]
```diff
@@ -5,8 +5,8 @@
 'use strict';
 
 var io = require('node-avro-io'),
-    avsc = require('../../../../lib');
-
+    avsc = require('../../../../lib'),
+    {Buffer} = require('buffer');
```
Comment (mtth) [inline etc/benchmarks/avro-serialization-implementations/scripts/decode/node-avro-io.js:9]: This syntax isn't compatible with older Node versions, could you swap to one that is? For example:

```js
    // ...
    buffer = require('buffer');

var Buffer = buffer.Buffer;
```

Also, feel free to skip the files in `etc/` as they are not included in the distributed package.

**Review by mtth — CHANGES_REQUESTED** [review]

File: etc/benchmarks/avro-serialization-implementations/scripts/decode/node-avro-io.js (hunk 1 above)
Comment (JAForbes) [reply etc/benchmarks/avro-serialization-implementations/scripts/decode/node-avro-io.js:9]: Yep no worries.  Yeah I think that syntax is Node 16+.

File: etc/browser/lib/crypto.js [hunk 2]
```diff
@@ -12,6 +12,9 @@
  *
  */
 
+var buffer = require('buffer');
+var Buffer = buffer.Buffer;
```
Comment (JAForbes) [inline etc/browser/lib/crypto.js:16]: @mtth I left this /etc file in, I wasn't sure if it was a polyfill for the browser and may need the explicit import.

Comment (mtth) [reply etc/browser/lib/crypto.js:16]: Good call, it is. I'd forgotten about that one.

**Review by mtth — APPROVED** [review]
Thanks for the PR, @JAForbes.

**Comment by mtth** [comment]
Published as `5.7.6`.

**Pull request merged**